observers/slack_resource.hpp

```diff
@@ -0,0 +1,95 @@
+#ifndef SERENITY_SLACK_RESOURCE_HPP
+#define SERENITY_SLACK_RESOURCE_HPP
+
+#include <mesos/mesos.hpp>
+
+#include <stout/result.hpp>
+
+#include <memory>
+#include <unordered_set>
+#include <string>
+#include <vector>
+
+#include "serenity/serenity.hpp"
+
+namespace mesos {
+namespace serenity {
+
+/**
+ * Struct that contains snapshot of metrics that concern us of previous
+ * sampling
+ */
+struct ExecutorSnapshot {
```

Comment:
I agree with Szymon that it's quite useful to use custom script then Resource Usage.

@skonefal  please, move that ExecutorSnapshot and cpu_usage calculation to some utils  - we'll need that in other components. And we will be able to change it in one place if needed (: